api/services/ocean/spark/paths/workspaces.yaml

```diff
@@ -0,0 +1,46 @@
+summary: Ocean Spark
+
+post:
+  summary: "Create Workspace"
+  description: |
+    Create Ocean Spark Workspace
+  operationId: "OceanSparkWorkspaceCreate"
+  tags:
+    - "Ocean Spark"
+  parameters:
+    - $ref: "../../../../commons/parameters/accountId.yaml"
+    - $ref: "../parameters/path/oceanSparkClusterId.yaml"
+  requestBody:
```

Comment:
the result is still incorrect : 
<img width="1838" alt="Screenshot 2024-04-12 at 15 22 43" src="https://github.com/spotinst/openapi/assets/21658174/d10eb971-0fb9-40cf-a7ff-84b9d2341074">